controllers/volumereplicationgroup_controller.go

```diff
@@ -951,20 +965,26 @@ func (v *VRGInstance) preparePVCForVRProtection(pvc *corev1.PersistentVolumeClai
 		// it is being deleted.
 		v.updateProtectedPVCCondition(pvc.Name, PVCProgressing, msg)
 
+		// No need to requeue, as a VRG reconcile request is expected when the
+		// PVC's state changes and becomes eligible.
 		return !requeue, skip
 	}
 
-	return v.protectPVC(pvc, log)
+	// PVC is eligible; prepare to protect it.
+	return v.prepareToProtectPVC(pvc, log)
 }
 
-func (v *VRGInstance) protectPVC(pvc *corev1.PersistentVolumeClaim,
+func (v *VRGInstance) prepareToProtectPVC(pvc *corev1.PersistentVolumeClaim,
```

Comment:
If above becomes `preparePVCForReplication` this possibly becomes `preparePVCForVRReplication`

Reply:
As I mentioned earlier, with the changes I made, I don't know see why this method applies only to VR.

Reply:
Currently we are VR specific, it is easier to reason on the present. When we get to the future we can look at renaming is more appropriately?